hivemq-edge/src/frontend/src/components/rjsf/MqttTransformation/components/DataModelDestination.tsx

```diff
@@ -0,0 +1,39 @@
+import { FC } from 'react'
+import { useTranslation } from 'react-i18next'
+import { JSONSchema7 } from 'json-schema'
+import { Card, CardBody, CardHeader, CardProps, Heading, HStack } from '@chakra-ui/react'
+
+import { useGetSubscriptionSchemas } from '@/api/hooks/useTopicOntology/useGetSubscriptionSchemas.tsx'
+import JsonSchemaBrowser from '@/components/rjsf/MqttTransformation/JsonSchemaBrowser.tsx'
+import LoaderSpinner from '@/components/Chakra/LoaderSpinner.tsx'
+import ValidationStatus from '@/components/rjsf/MqttTransformation/components/mapping/ValidationStatus.tsx'
+import { MappingValidation } from '@/modules/Subscriptions/types.ts'
+
+interface DataModelDestinationProps extends CardProps {
+  topic: string | undefined
+  validation: MappingValidation
+}
+
+const DataModelDestination: FC<DataModelDestinationProps> = ({ topic, validation, ...props }) => {
+  const { t } = useTranslation('components')
+  const { data, isLoading } = useGetSubscriptionSchemas(topic as string, topic ? 'destination' : undefined)
+
+  const isReady = Boolean(!isLoading && data)
+
+  return (
+    <Card {...props} size="sm">
+      <CardHeader as={HStack} justifyContent="space-between">
+        <Heading as="h3" size="sm">
+          {t('rjsf.MqttTransformationField.destination.header')}
+        </Heading>
+        <ValidationStatus validation={validation} />
+      </CardHeader>
+      <CardBody>
+        {isLoading && <LoaderSpinner />}
+        {isReady && <JsonSchemaBrowser schema={data as JSONSchema7} />}
```

Comment:
Not sure that this case could potentially happen, but theoretically, you'd get an empty card here if you loading is finished but there's no data.
I would argue that an error message if this happened would be appropriate.